airmozilla/manage/tests/views/test_vidlymedia.py

```diff
@@ -83,8 +81,7 @@
 </Response>
 """
 
-SAMPLE_MEDIA_RESULT_SUCCESS = """
-<?xml version="1.0"?>
+SAMPLE_MEDIA_RESULT_SUCCESS = """<?xml version="1.0"?>
```

Comment:
All of these changes were unnecessary. Note how there's a `""".strip()` at the end. This way the first line can be after a nice little newline.